### DISCUSSION TITLE
传输速度较慢

### DISCUSSION THREAD

**Discussion by Jake112222** [localsend/localsend#1246] [L-简体中文]
![微信截图_20240330142914](https://github.com/localsend/localsend/assets/165458943/97a495c1-6d08-462f-bb1f-88177b0c3dbd)
这是怎么回事

**Comment by tiny-andr** [comment]
在互传的两个设备中，高级设置里面关闭加密试试。我关闭之后直接5倍提升。

**Reply by Jake112222** [reply]
关掉之后有提升虽然没有你说得5倍提升，但是还是非常感谢
